tabular/src/autogluon/tabular/learner/abstract_learner.py

```diff
@@ -302,6 +303,8 @@ def predict_proba_multi(
         inverse_transform : bool, default = True
             If True, will return prediction probabilities in the original format.
             If False (advanced), will return prediction probabilities in AutoGluon's internal format.
+        use_refit_parent_oof: bool = True
```

Comment:
unrelated but `_post_process_predict()` and `_post_process_predict_proba()` is what we do in TabRepo's abstract class as well if my understanding is correct?

Reply:
yes, pretty much